### ISSUE TITLE
write table descriptions for complex tables, and titles for all tables

### ISSUE THREAD

**Issue by snotskie** [Tiny-Earth/DemoSite#12]
For example: https://github.com/Tiny-Earth/DemoSite/blob/4e507465ae84787f1695f3432c9bf4cf18581e18/site/src/experiences.md?plain=1#L270

Note, very few, if any, tables should require a table description for accessibility. However, those that convey a message that vision-impared users would be at a disadvantage to parse should have a description that makes the story of the table explicit.

For those that don't require a description, remove the description placeholder

And all tables need an informative, short, unique title

**Comment by snotskie** [comment]
444052f0423988e761976f6a3b26baa1677e3438
